src/server/modes/base/maintenance/players/connect.ts

```diff
@@ -326,6 +340,11 @@ export default class GamePlayersConnect extends System {
     this.emit(BROADCAST_SCORE_BOARD, connectionId);
     this.emit(RESPONSE_SEND_PING, connectionId);
     this.emit(PLAYERS_APPLY_SHIELD, player.id.current, PLAYERS_SPAWN_SHIELD_DURATION_MS);
+
+    if (player.user) {
```

Comment:
It's better to use ’hasOwnProperty’ (here and next) to not to get unexpected result. There is `has` function in `src/support/objects.ts`.